js/src/components/modals/SEMrushCountrySelector.js

```diff
@@ -155,7 +156,7 @@ const countries = [
 /**
  * The SEMrush Country Selector component.
  */
-class SemRushCountrySelector extends Component {
+class SEMrushCountrySelector extends Component {
```

Comment:
I can't recall if we agreed to this capitalization: if so, we should then rename all the other components...